cmake/projects/SPIRV-Headers/hunter.cmake

```diff
@@ -43,6 +43,28 @@ hunter_add_version(
     e1666e3cf71e0eb8c75a1786e9f430f6d7670f42
 )
 
+hunter_add_version(
+    PACKAGE_NAME
+    SPIRV-Headers
+    VERSION
+    1.5.4
+    URL
+    https://github.com/KhronosGroup/SPIRV-Headers/archive/1.5.4.tar.gz
+    SHA1
+    24e517cc55339b847ccb7d3b1d59f73b7df775a1
+)
+
+hunter_add_version(
+    PACKAGE_NAME
+    SPIRV-Headers
+    VERSION
+    1.5.4.raytracing.fixed
```

Comment:
Can you set the default config to use this latest version?

Reply:
I'd suggest to wait util we bump up SPIRV-Tools to v2020.6. SPIRV-Tools are really picky about SPIRV-Headers version and their versions needs to be matched exactly.

Reply:
We still need to test this version, so let's update the default, check CI, and then downgrade if necessary.

Reply:
done